src/LIBMONGOCRYPT_VERSION_CURRENT

```diff
@@ -1 +1 @@
-1.14.1
+1.15.2-20250912+git03cba710f7
```

Comment:
Do we depend on the development version of libmongocrypt 1.15.2? Would it make sense to wait for a release?

Reply:
Originally opened this before the release. It has since been bumped to the 1.15.2 tag, though.